python/Ganga/Core/GangaRepository/SubJobXMLList.py

```diff
@@ -232,7 +232,10 @@ def __get_dataFile(self, index, force_backup=False):
     def __len__(self):
         """ return length or lookup the last modified time compare against self._stored_len[0] and if nothings changed return self._stored_len[1]"""
         import os
-        this_time = os.stat(self._jobDirectory).st_ctime
+        try:
+            this_time = os.stat(self._jobDirectory).st_ctime
+        except OSError:
+            return 0
```

Comment:
This doesn't seem related to this change at all? I think this is from one of your other PRs? Could we keep the PR clean please?